js/about/styles.js

```diff
@@ -312,6 +312,13 @@ class AboutStyle extends ImmutableComponent {
           &lt;BrowserButton iconOnly icon={'{'}globalStyles.appIcons.private{'}'} size='60px' color='red' />
         </Code></Pre>
 
+        <BrowserButton groupedItem secondaryColor panelItem l10nId='panelItem' onClick={this.onEnableAutoplay} />
+        <BrowserButton groupedItem primaryColor panelItem l10nId='panelItem' onClick={this.onEnableAutoplay} />
```

Comment:
Maybe `this.copyToClipboard` and `this.props.showQRcode` look better, which are actually used only on bitcoinDashboard.js for now.